springwolf-examples/springwolf-kafka-example/src/main/java/io/github/stavshamir/springwolf/example/configuration/AsyncApiConfiguration.java

```diff
@@ -59,4 +65,59 @@ public AsyncApiDocket asyncApiDocket() {
                 .build();
     }
 
+    private static AsyncHeaders createSpringDefaultHeaders() {
```

Comment:
We should consider providing factory methods for common uses like spring type id header and cloud events in `AsyncHeaders`

Reply:
We also thought that this might be a good idea and updated the PR accordingly. 
The builder for Spring headers are in the spring kafka extension.  
The builder for Cloud Events is located in core, because Cloud Event headers are more general and not limited to Kafka (or another messaging system) as long as headers are supported.
The Kafka example has been updated accordingly.